recipes/avahi-system/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SysConfigAvahiSystemConan(ConanFile):
+    name = "avahi-system"
+    version = "system"
+    description = "Conan package for system-installed Avahi - Service Discovery for Linux using mDNS/DNS-SD - compatible with Bonjour"
+    topics = ("Avahi", "Bonjour", "DNS-SD", "mDNS")
```

Comment:
Would adding `provides` help prevent conflicts with the other libraries?

Reply:
Good idea. We'd need to make one of the packages canonical... Maybe the `mdnsresponder` one... though `dns_sd` or `libdns_sd` would be the idea name perhaps.

Reply:
Done, #6928 and #6930 both state `provides = "mdnsresponder"` to align with #6937.